content/doc/advanced/hacking-on-gitea.zh-cn.md

```diff
@@ -15,7 +15,7 @@ menu:
 
 # Hacking on Gitea
 
-首先你需要一些运行环境，这和 [从源代码安装](zh-cn/installation/from-source) 相同，如果你还没有设置好，可以先阅读那个章节。
```

Comment:
This linking is generally simply wrong, please fix the link based on https://gohugo.io/extras/crossreferences/